service/src/main/java/bio/terra/catalog/rawls/RawlsException.java

```diff
@@ -0,0 +1,15 @@
+package bio.terra.catalog.rawls;
+
+import bio.terra.common.exception.ErrorReportException;
+import bio.terra.rawls.client.ApiException;
+import org.springframework.http.HttpStatus;
+
+public class RawlsException extends ErrorReportException {
+  public RawlsException(String message, ApiException e) {
+    super(message, e, null, HttpStatus.resolve(e.getCode()));
+  }
+
+  public RawlsException(ApiException e) {
+    super(e, HttpStatus.resolve(e.getCode()));
```

Comment:
According to coverage, this line isn't hit by any tests. Is it used in the code? If not, I would remove it until it's needed.